control/lti.py

```diff
@@ -413,24 +458,33 @@ def damp(sys, doprint=True):
                       (p.real, p.imag, d, w))
     return wn, damping, poles
 
-def evalfr(sys, x):
+def evalfr(sys, x, squeeze=True):
     """
-    Evaluate the transfer function of an LTI system for a single complex
-    number x.
+    Evaluate the transfer function of an LTI system for complex frequency x.
+
+    Returns the complex frequency response `sys(x)` where `x` is `s` for
+    continuous-time systems and `z` for discrete-time systems.
 
-    To evaluate at a frequency, enter x = omega*j, where omega is the
-    frequency in radians
+    To evaluate at a frequency omega in radians per second, enter
+    ``x = omega * 1j`` for continuous-time systems, or
+    ``x = exp(1j * omega * dt)`` for discrete-time systems, or use
+    ``freqresp(sys, omega)``.
 
     Parameters
     ----------
     sys: StateSpace or TransferFunction
         Linear system
-    x: scalar
-        Complex number
+    x: complex scalar or array_like
+        Complex frequency(s)
+    squeeze: bool, optional (default=True)
+        If True and sys is single input single output (SISO), returns a
+        1D array or scalar depending on the length of x.
```

Comment:
```suggestion
    x : complex scalar or array_like
        Complex frequency(s)
    squeeze : bool, optional (default=True)
        If True and `sys` is single input single output (SISO), i.e. `m=1`,
        `p=1`, return a 1D array rather than a 3D array.
```